Makefile

```diff
@@ -183,3 +183,19 @@ clean:
 		cover.out oci-archives bundle bundle.Dockerfile \
 		skupper-*.tgz artifacthub-repo.yml \
 		network-observer-*.tgz  skupper-*-scope.yaml
+
+# Run a specific test
+test:
```

Comment:
A `test` targe already exists. I believe this one can be removed as I do not see
an advantage of keeping it, considering that you also introduced `test-subset`.

Reply:
DOne under: 4cf39555a81086798f6bb787ceef1354227a4224